source/jormungandr/tests/kirin_realtime_tests.py

```diff
@@ -582,7 +582,7 @@ def test_vj_delay_day_after(self):
             '20120615T070436',
             '20120615T070520',
         ]  # pt_walk + rt 07:02:24
-        assert get_used_vj(day_after_response), [[] == ['vjA:modified:0:96231_2015-07-28_0']]
+        assert get_used_vj(day_after_response), [[] == ['vehicle_journey:vjA:modified:0:96231_2015-07-28_0']]
```

Comment:
Okay, if I understand, the main reason is for being ISO with the rest of the code

Reply:
Yep, RT-VJ having their uris prefixed the same way.